homework/index.js

```diff
@@ -3,7 +3,9 @@
 {
   function fetchJSON(url, cb) {
     const xhr = new XMLHttpRequest();
+    // "Authorization: token f55126d198f3c6e9e7ef74e04919e059b82a231b"
```

Comment:
Hi Ramazan - you don't need to have this comment here 🙂

Reply:
Let me delete and see what will happen.

Reply:
We added that line for us to  browse when many users try to enter the web site

Reply:
🚨Please @RamazanInce DELETE this token from your authorisation list. This is a security concern.

Reply:
I changed it add push it again.
that will be okey?
thanks

Reply:
Not only in the code, it has to be deactivated from your GitHub account, Go to https://github.com/settings/tokens and delete it.

Reply:
Let me do it in that way